.github/packaging/SharpVector.nuspec

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package >
+    <metadata>
+        <id>SharpVector</id>
+        <version>1.0.0</version>
+        <description>
+            SharpVector utility - std::vector with all .NET's List features: ForEach, FindAll, and more!
+            Increase your productivity and vector handling with just one header.
+        </description>
+        <authors>BartoszKlonowski</authors>
+        <tags>Native native vector Cpp List ForEach FindAll BinarySearch Exists CopyTo Find IndexOf</tags>
+        <license type="file">LICENSE.txt</license>
+        <projectUrl>https://github.com/BartoszKlonowski/SharpVector</projectUrl>

```

Comment:
Project's repository path will have to be updated:
*SharpVector* -> *ExtendedVector*

Applies to all *SharpVector* occurrences in this file.